sdk/generic-sdk-core/generic-core/src/main/java/com/generic/core/util/Context.java

```diff
@@ -146,11 +117,7 @@ public Object getValue() {
      * @throws IllegalArgumentException If {@code key} is {@code null}.
      */
     public Context addData(Object key, Object value) {
```

Comment:
Do we want to also support `remove`?

Reply:
Let's follow-up on this in another potential change. Was hoping to keep this change to simply restructuring in how Context is stored in memory and the small renamings and removal of Optional